core/arch/arm/kernel/entry_a32.S

```diff
@@ -551,10 +553,19 @@ shadow_stack_access_ok:
 	ldr	r0, =threads
 	ldr	r0, [r0, #THREAD_CTX_STACK_VA_END]
 	mov	sp, r0
+	bl	__get_core_pos
+	mov	r1, #THREAD_CORE_LOCAL_SIZE
+	ldr	r2, =thread_core_local
+	mla	r8, r0, r1, r2
+	ldr	r10, [r8, #THREAD_CORE_LOCAL_FLAGS]
+	bic	r10, r10, #THREAD_CLF_TMP
+	str	r10, [r8, #THREAD_CORE_LOCAL_FLAGS]
```

Comment:
Or perhaps:
```suggestion
	mov	r10, #0
	str	r10, [r8, #THREAD_CORE_LOCAL_FLAGS]
```
At this point it seems `flags` can only be `== THREAD_CLF_TMP` with no other bit set, or am I missing something?

Reply:
Agree

Reply:
Fixed